src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -129,7 +129,7 @@ namespace Platform
         {
             log_fatal("failed to read /proc/self/exe");
         }
-#    elif defined(__FreeBSD__)
+#    elif defined(__FreeBSD__) || defined(__NetBSD__)
         const int32_t mib[] = { CTL_KERN, KERN_PROC, KERN_PROC_PATHNAME, -1 };
```

Comment:
NetBSD needs different parameters here.

Reply:
Can you elaborate? Which parameters does it need?

Reply:
https://nxr.netbsd.org/xref/src/lib/libexecinfo/backtrace.c#63

Reply:
So if I understand correctly, on NetBSD, this line should read as follows:
```C++
const int32_t mib[] =  { CTL_KERN, KERN_PROC_ARGS, -1, KERN_PROC_PATHNAME };
```
?

Reply:
Yes.